### PR TITLE
Update importance feature for CS2040CFile, Specify Case sensitivity and Documents accepted in UG

### PR DESCRIPTION
**Pull request by nicholas132000**

### CODE REVIEW

File: src/main/java/seedu/clialgo/command/AddCommand.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * The <code>AddCommand</code> objects represents the user command to add new CS2040CFiles into CLIAlgo.
  */
 public class AddCommand extends Command {
-    public static final int DEFAULT_IMPORTANCE = 5;
+    public static final int DEFAULT_IMPORTANCE = 0;
```
Comment (heejet) [inline src/main/java/seedu/clialgo/command/AddCommand.java:16]: why is this 0?

File: src/main/java/seedu/clialgo/command/FilterByImportanceCommand.java [hunk 2]
```diff
@@ -46,8 +46,14 @@ public void printAllTopics(TopicManager topicManager, Ui ui) {
...
+                System.out.println(serialNumber + ". " + cs2040cFileType + cs2040cFile.getName() + " [" +
+                        "no importance assigned to this CS2040CFile" + "]");
+                ++serialNumber;
+            } else {
+                System.out.println(serialNumber + ". " + cs2040cFileType + cs2040cFile.getName() + " [" +
+                        cs2040cFile.getImportance() + "]");
+            }
             ++serialNumber;
         }
         ui.printDivider();
```
Comment (heejet) [inline src/main/java/seedu/clialgo/command/FilterByImportanceCommand.java:59]: The idea behind the issue is that our UG did not mention that leaving the `importance` field empty sets it to `DEFAULT_IMPORTANCE`. So to resolve it we just need to add it to our UG and make it clear to the user not change our implementation.

**Review by heejet — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/clialgo/command/FilterByImportanceCommand.java (hunk 2 above)
Comment (nicholas132000) [reply src/main/java/seedu/clialgo/command/FilterByImportanceCommand.java]: Made the necessary changes;)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -106,22 +106,24 @@ Valid TOPIC's are 'SORTING', 'LINKED_LIST', 'GRAPH_STRUCTURES',
...
 `.jar` file.
 - Not following the syntax strictly would result in an invalid command message.
+- `n/` and `t/` fields must be non-empty.
 - Only full words will be matched. 
   - For example, `LINKED_LIST` will not match `Link List`.
-- `n/` and `t/` fields must be non-empty.
-- `i/` field is optional. If this field is left empty, a note or code files' importance is set to 5 (default).
+- `/t` Topic field is **case-sensitive**. Enter `help c/add` to view list of valid topics and their case-sensitivities.
+  - For example, keying in `add n/queue t/sorting` is **not** valid. It should be `add n/queue t/SORTING` instead.
+- `i/` field is optional. **If this field is left empty, the note or code files' importance is set to 5 by default**.
```
Comment (heejet) [inline docs/UserGuide.md:126]: Siuuuu

**Review by heejet — APPROVED** [review]

**Pull request merged**